src/metabase/util.cljc

```diff
@@ -202,22 +204,26 @@
 (def ^{:arglists '([x])} ->kebab-case-en
   "Like [[camel-snake-kebab.core/->kebab-case]], but always uses English for lower-casing, supports keywords with
   namespaces, and returns `nil` when passed `nil` (rather than throwing an exception)."
-  (wrap-csk-conversion-fn-to-handle-nil-and-namespaced-keywords ->kebab-case-en*))
+  (wrap-csk-conversion-fn-to-handle-nil-and-namespaced-keywords
+   (memoize/lru ->kebab-case-en* :lru/threshold 256)))
```

Comment:
We'll only ever pass the same set of perhaps a few hundred values to this - it seems like we could save the LRU overhead and use `clojure.core/memoize` to cache everything forever.

Reply:
I'll need more convincing. (I'm old enough to have lived in a world where Prolog systems had no GC for atoms and felt the pain.)